README.md

```diff
@@ -6,6 +6,12 @@
 MedCo documentation is centralized on the following website: 
 [MedCo Unlynx](https://medco.epfl.ch/documentation/developer/components/medco-unlynx.html).
 
+## Version
+
+We have a development and a stable version. The `master`-branch in `github.com/lca1/medco-unlynx` is the development version that works but can have incompatible changes.
+
+Use one of the latest tags `v0.1.3` that are stable and have no incompatible changes.
```

Comment:
I would say to not put the version number in here (so that we don't forget everytime to update), and just mentionning latest release.

Reply:
Agree